src/rpdk/core/test.py

```diff
@@ -133,6 +167,8 @@ def setup_subparser(subparsers, parents):
         "--role-arn", help="Role used when performing handler operations."
     )
 
+    parser.add_argument("--endpoint-url", help="CloudFormation endpoint to use.")
```

Comment:
There's also an endpoint argument for contract tests I would make both a little more explicit in the name so that people don't get confused

https://github.com/aws-cloudformation/cloudformation-cli/blob/33a7fa4ad70b8fe52b635ed19f173f363c054356/src/rpdk/core/test.py#L139-L162

Reply:
Hm ok, I was mimicking the standard for specifying service endpoints in aws cli so I thought that might be clear enough but I can make it more explicit